<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>831</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>安全笔记本</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/png/notepad.png</normaloff>:/png/notepad.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBoxTitle">
      <property name="maximumSize">
       <size>
        <width>256</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>目录(&amp;H)</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeView" name="treeView">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxNote">
      <property name="title">
       <string>笔记(&amp;N)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0" colspan="2">
        <widget class="MTextEdit" name="textEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="lineEditTitle"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1044</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionOpenOnline"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑(&amp;E)</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionCut"/>
    <addaction name="separator"/>
    <addaction name="actionSearch"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuNote">
    <property name="title">
     <string>笔记(&amp;N)</string>
    </property>
    <addaction name="actionAddPartition"/>
    <addaction name="actionRemovePartition"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionDelete"/>
    <addaction name="actionEdit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>工具(&amp;T)</string>
    </property>
    <addaction name="actionGenPassword"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="title">
     <string>格式(O)</string>
    </property>
    <addaction name="actionTextBold"/>
    <addaction name="actionTextItalic"/>
    <addaction name="actionTextUnderline"/>
    <addaction name="actionStrikeout"/>
    <addaction name="separator"/>
    <addaction name="actionAlignLeft"/>
    <addaction name="actionAlignCenter"/>
    <addaction name="actionAlignRight"/>
    <addaction name="actionAlignJustify"/>
    <addaction name="separator"/>
    <addaction name="actionIndentLess"/>
    <addaction name="actionIndentMore"/>
    <addaction name="separator"/>
    <addaction name="actionTextColor"/>
    <addaction name="actionTextBackgroundColor"/>
    <addaction name="separator"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionZoomIn"/>
    <addaction name="actionToggleCheckState"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuFormat"/>
   <addaction name="menuNote"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarFile">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionOpenOnline"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBarEdit">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
  </widget>
  <widget class="QToolBar" name="toolBarPartition">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRemovePartition"/>
   <addaction name="actionAddPartition"/>
  </widget>
  <widget class="QToolBar" name="toolBarSettings">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QToolBar" name="toolBarRichEditor">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionCut"/>
   <addaction name="separator"/>
   <addaction name="actionTextBold"/>
   <addaction name="actionTextItalic"/>
   <addaction name="actionTextUnderline"/>
   <addaction name="actionStrikeout"/>
   <addaction name="separator"/>
   <addaction name="actionAlignLeft"/>
   <addaction name="actionAlignCenter"/>
   <addaction name="actionAlignRight"/>
   <addaction name="actionAlignJustify"/>
   <addaction name="separator"/>
   <addaction name="actionIndentLess"/>
   <addaction name="actionIndentMore"/>
   <addaction name="separator"/>
   <addaction name="actionTextColor"/>
   <addaction name="actionTextBackgroundColor"/>
   <addaction name="separator"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionZoomIn"/>
  </widget>
  <widget class="QToolBar" name="toolBarFont">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/new.png</normaloff>:/png/new.png</iconset>
   </property>
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
   <property name="toolTip">
    <string>新建笔记本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/open.png</normaloff>:/png/open.png</iconset>
   </property>
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
   <property name="toolTip">
    <string>打开现有的笔记本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/save1.png</normaloff>:/png/save1.png</iconset>
   </property>
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>保存笔记本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/save_as.png</normaloff>:/png/save_as.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>笔记本另存为</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/exit1.png</normaloff>:/png/exit1.png</iconset>
   </property>
   <property name="text">
    <string>退出(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>退出笔记本</string>
   </property>
   <property name="shortcut">
    <string>Alt+F4</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/exit.png</normaloff>:/png/exit.png</iconset>
   </property>
   <property name="text">
    <string>关闭(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>关闭笔记本</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/copy.png</normaloff>:/png/copy.png</iconset>
   </property>
   <property name="text">
    <string>复制(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>复制</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/paste.png</normaloff>:/png/paste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>粘贴</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/cut.png</normaloff>:/png/cut.png</iconset>
   </property>
   <property name="text">
    <string>剪切(&amp;X)</string>
   </property>
   <property name="toolTip">
    <string>剪切</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/settings.png</normaloff>:/png/settings.png</iconset>
   </property>
   <property name="text">
    <string>设置(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>笔记本设置</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/About.png</normaloff>:/png/About.png</iconset>
   </property>
   <property name="text">
    <string>关于(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>关于笔记本</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/qt.png</normaloff>:/png/qt.png</iconset>
   </property>
   <property name="text">
    <string>关于QT</string>
   </property>
   <property name="toolTip">
    <string>关于QT</string>
   </property>
  </action>
  <action name="actionSearch">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/search.png</normaloff>:/png/search.png</iconset>
   </property>
   <property name="text">
    <string>查找(&amp;F)</string>
   </property>
   <property name="toolTip">
    <string>查找笔记</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/add.png</normaloff>:/png/add.png</iconset>
   </property>
   <property name="text">
    <string>添加(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>添加笔记</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/Delete1.png</normaloff>:/png/Delete1.png</iconset>
   </property>
   <property name="text">
    <string>删除(&amp;D)</string>
   </property>
   <property name="toolTip">
    <string>删除笔记</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/edit.png</normaloff>:/png/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑(&amp;E)</string>
   </property>
   <property name="toolTip">
    <string>编辑笔记</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAddPartition">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/folder_plus.png</normaloff>:/png/folder_plus.png</iconset>
   </property>
   <property name="text">
    <string>添加分区(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>添加分区</string>
   </property>
  </action>
  <action name="actionRemovePartition">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/folder_minus.png</normaloff>:/png/folder_minus.png</iconset>
   </property>
   <property name="text">
    <string>删除分区(&amp;R)</string>
   </property>
   <property name="toolTip">
    <string>删除当前分区, 子分区和笔记会被删除</string>
   </property>
  </action>
  <action name="actionOpenOnline">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/download_from_clud.png</normaloff>:/png/download_from_clud.png</iconset>
   </property>
   <property name="text">
    <string>打开在线文档(&amp;L)</string>
   </property>
  </action>
  <action name="actionGenPassword">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/vault.png</normaloff>:/png/vault.png</iconset>
   </property>
   <property name="text">
    <string>生成密码(&amp;P)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionTextBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-text-bold-symbolic.symbolic.png</normaloff>:/png/format-text-bold-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
   <property name="toolTip">
    <string>加粗</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionTextItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-text-italic-symbolic.symbolic.png</normaloff>:/png/format-text-italic-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
   <property name="toolTip">
    <string>斜体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionTextUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-text-underline-symbolic.symbolic.png</normaloff>:/png/format-text-underline-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
   <property name="toolTip">
    <string>下划线</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionStrikeout">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-text-strikethrough-symbolic.symbolic.png</normaloff>:/png/format-text-strikethrough-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>删除线</string>
   </property>
   <property name="toolTip">
    <string>删除线</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionAlignLeft">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-justify-left-symbolic.symbolic.png</normaloff>:/png/format-justify-left-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="toolTip">
    <string>左对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionAlignCenter">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-justify-center-symbolic.symbolic.png</normaloff>:/png/format-justify-center-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐</string>
   </property>
   <property name="toolTip">
    <string>居中对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionAlignRight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-justify-right-symbolic.symbolic.png</normaloff>:/png/format-justify-right-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="toolTip">
    <string>右对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionf_list_bullet">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/view-list-symbolic.symbolic.png</normaloff>:/png/view-list-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>项目符号</string>
   </property>
   <property name="toolTip">
    <string>项目符号</string>
   </property>
  </action>
  <action name="actionf_list_ordered">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/list-remove-all-symbolic.symbolic.png</normaloff>:/png/list-remove-all-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>项目编号</string>
   </property>
   <property name="toolTip">
    <string>项目编号</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom-in-symbolic.symbolic.png</normaloff>:/png/zoom-in-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>加大字体</string>
   </property>
   <property name="toolTip">
    <string>加大字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/zoom-out-symbolic.symbolic.png</normaloff>:/png/zoom-out-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>减小字体</string>
   </property>
   <property name="toolTip">
    <string>减小字体</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionTextColor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-font-color.png</normaloff>:/png/format-font-color.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
   <property name="toolTip">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="actionTextBackgroundColor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-font-color-front.png</normaloff>:/png/format-font-color-front.png</iconset>
   </property>
   <property name="text">
    <string>字体背景颜色</string>
   </property>
   <property name="toolTip">
    <string>字体背景颜色</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/edit-undo-symbolic.symbolic.png</normaloff>:/png/edit-undo-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>撤销</string>
   </property>
   <property name="toolTip">
    <string>撤销</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/edit-redo-symbolic.symbolic.png</normaloff>:/png/edit-redo-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>重做</string>
   </property>
   <property name="toolTip">
    <string>重做</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionAlignJustify">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-justify-fill-symbolic.symbolic.png</normaloff>:/png/format-justify-fill-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>两端对齐</string>
   </property>
   <property name="toolTip">
    <string>两端对齐</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionToggleCheckState">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/object-select-symbolic.symbolic.png</normaloff>:/png/object-select-symbolic.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>Chec&amp;ked</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+K</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionIndentLess">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-indent-less-symbolic-rtl.symbolic.png</normaloff>:/png/format-indent-less-symbolic-rtl.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>缩进</string>
   </property>
   <property name="toolTip">
    <string>缩进</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
  <action name="actionIndentMore">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/png/format-indent-more-symbolic-rtl.symbolic.png</normaloff>:/png/format-indent-more-symbolic-rtl.symbolic.png</iconset>
   </property>
   <property name="text">
    <string>取消缩进</string>
   </property>
   <property name="toolTip">
    <string>取消缩进</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
   <property name="priority">
    <enum>QAction::LowPriority</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MTextEdit</class>
   <extends>QTextEdit</extends>
   <header>mtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
